src/pages/Post.js

```diff
@@ -50,6 +50,9 @@ class PostPage extends React.Component {
       status: null,
       seniorClassYear: 2020,
       filters: {
+        toggle: {
+          enabled: false
+        },
```

Comment:
Does this belong here? I think it makes more sense to place it outside the filters dictionary and rename it as a single boolean field filtersEnabled (i.e. ```this.state.filtersEnabled```).